test/integration/getFunctions.v2.test.ts

```diff
@@ -0,0 +1,84 @@
+import {Client} from '../../src/client/client';
+import {GetAuthorizationHeader} from '../../src/client/clientConfig';
+
+const accountId = process.env['ACCOUNT_ID'] || 'does-not-exist';
+const userId = process.env['USER_ID'] || 'does-not-exist';
+const bearer = process.env['BEARER'] || 'does-not-exist';
+
+describe('Get Functions V2', () => {
+  it('should get V2 functions', async () => {
+    // custom auth implementation start
+    const getAuthorizationHeader: GetAuthorizationHeader = async ({
+      url,
+      method,
+    }) => {
+      return new Promise((resolve, reject) => {
+        resolve(`Bearer ${bearer}`);
+      });
+    };
+    const client = new Client({
+      accountId,
+      authStrategy: getAuthorizationHeader,
+      failOnErrorStatusCode: true,
+    });
+
+    const response = await client.getFunctions({
+      lpEventSource: 'tests',
+      userId,
+    });
+
+    expect(response.ok).toEqual(true);
+    expect(Array.isArray(response.body)).toEqual(true);
```

Comment:
Should we check here also the format of the functions? 

After all, that's most relevant for our end users

Reply:
Or did you do that in the unit tests ?

Reply:
the lambdas are contained in the response body and is a an array, but dont see the need to check the properties since there is no type for it , i just check that is an array 
![image](https://github.com/user-attachments/assets/facf2d34-3abc-454c-9400-3203b9091e52)

Reply:
GetFunctions It is whatever FM returns , properties. format  is not relevant for the client, it just returns it in the resposne.body